mbed-trace/mbed_trace.h

```diff
@@ -108,9 +108,9 @@ extern "C" {
 #define tr_error(...)           mbed_tracef(TRACE_LEVEL_ERROR,   TRACE_GROUP, __VA_ARGS__)   //!< Print Error Message
 #define tr_err(...)             mbed_tracef(TRACE_LEVEL_ERROR,   TRACE_GROUP, __VA_ARGS__)   //!< Alternative error message
 #define tr_cmdline(...)         mbed_tracef(TRACE_LEVEL_CMD,     TRACE_GROUP, __VA_ARGS__)   //!< Special print for cmdline. See more from TRACE_LEVEL_CMD -level
-
-/** Possible to skip all traces in compile time */
-#if defined(YOTTA_CFG_MBED_TRACE) || (defined(YOTTA_CFG) && !defined(NDEBUG))
+//aliases for the most commonly used functions
+#define tracef(dlevel, grp, ...)    mbed_tracef(dlevel, grp, __VA_ARGS__)   //!< Alias for mbed_tracef()
+#define trace_array(buf, len)       mbed_trace_array(buf, len)              //!< Alias for mbed_tracef()
```

Comment:
Would seem logical. (But as an addition for now, to avoid immediate respellings.)